code/modules/shuttle/dropship_hijack.dm

```diff
@@ -126,9 +127,14 @@
 					to_chat(M, SPAN_DANGER("You feel the ship turning sharply as it adjusts its course!"))
 					shake_camera(M, 60, 2)
 			playsound_area(internal_area, 'sound/effects/antiair_explosions.ogg')
+		addtimer(CALLBACK(src, PROC_REF(aa_interception), shuttle), 5 SECONDS)
 
 	hijacked_bypass_aa = TRUE
 
+/datum/dropship_hijack/almayer/proc/aa_interception(/obj/docking_port/mobile/shuttle)
+	shuttle.intoTheSunset()
+	SSticker.mode.round_finished = MODE_INFESTATION_X_MINOR
```

Comment:
This should be handled by the gamemode `check_win` proc. Consider setting maybe `SShijack.hijack_status` to a hijack_averted or something, and then `/datum/game_mode/colonialmarines/check_win()` can check that value and end the round.

As is, this means every game mode that shoots down the dropship successfully will abruptly end in distress signal xeno minor.

Reply:
No longer relevant.